src/include/migraphx/op/scatternd_op.hpp

```diff
@@ -28,44 +28,87 @@
 #include <migraphx/check_shapes.hpp>
 #include <migraphx/argument.hpp>
 #include <migraphx/par_for.hpp>
+#include <migraphx/ranges.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace op {
-
+/**
+ * @brief
+ * N-dimensional Scatter operations. This struct is parent class to ops which differ in what formula
+ * is used to reduce (combine old and new values of) the scattered value.  It is an implementation
+ * of the Onnx ScatterND operation (see
```

Comment:
I dont think we should reference frontend operators as they can diverge. Also this could be applied to [`tf.scatter_nd`](https://www.tensorflow.org/api_docs/python/tf/scatter_nd), so its not onnx specific.

Reply:
The real point of the comment is that the documentation for the Onnx operator serves as the specification for its behavior.  How would you want to say that?  It would be nice if AMD had its own specification, but I don't think we do.

Reply:
comment changed.